### PR TITLE
Sidebar fixes: Add on-prem connector setup and fix links from 2 API product pages

### LINKED ISSUE #74: On-premise connector setup article missing from nav
Following docs page is missing from Integrations > Accounting section of the side nav.

It should be added after Zoho Books or to another appropriate place.

https://docs.codat.io/docs/offline-connectors

### PR DESCRIPTION
**Pull request by d-laing**
# Description

Fixes #74 

image

Also fixes a couple of broken links from the Banking API and Commerce API homepages, which were missing the initial `/`s.

## Type of change

Please delete options that are not relevant.

- [x] Fixes

### CODE REVIEW

File: sidebars/commerce-api.js [hunk 1]
```diff
@@ -9,16 +9,16 @@ module.exports = [
...
   },
   {
     type: "link",
     label: "Commerce integrations",
-    href: "integrations/commerce/overview",
+    href: "/integrations/commerce/overview",
   },
   {
     type: "link",
-    href: "/commerce-api",
```
Comment (d-laing) [inline sidebars/commerce-api.js:21]: I reordered these attributes to match the others.

File: sidebars/commerce-api.js [hunk 2]
```diff
@@ -9,16 +9,16 @@ module.exports = [
...
     type: "link",
     label: "Commerce data model",
-    href: "data-model/commerce/",
+    href: "/data-model/commerce/",
   },
   {
     type: "link",
     label: "Commerce integrations",
-    href: "integrations/commerce/overview",
+    href: "/integrations/commerce/overview",
```
Comment (mcclowes) [inline sidebars/commerce-api.js:17]: Pretty sure all of these should not have the proceeding `/`

**Review by mcclowes — CHANGES_REQUESTED** [review]

File: sidebars/commerce-api.js (hunk 2 above)
Comment (mcclowes) [reply sidebars/commerce-api.js:17]: Throughout all of the sidebar pretty much

Comment (d-laing) [reply sidebars/commerce-api.js:17]: Hmm, OK. These links are currently broken though, e.g. https://codat-docs.vercel.app/commerce-api/overview

Commerce data model
Commerce integrations

Comment (mcclowes) [reply sidebars/commerce-api.js:17]: I was wrong, sorry!

**Review by mcclowes — APPROVED** [review]

**Pull request merged**